docs/reference/scripting/using.asciidoc

```diff
@@ -1,425 +1,448 @@
 [[modules-scripting-using]]
-== How to use scripts
+== How to write scripts
 
-Wherever scripting is supported in the Elasticsearch API, the syntax follows
-the same pattern:
+Wherever scripting is supported in the {es} APIs, the syntax follows the same
+pattern; you specify the language of your script, provide the script logic (or
+source, and add parameters that are passed into the script:
 
 [source,js]
 -------------------------------------
   "script": {
-    "lang":   "...",  <1>
-    "source" | "id": "...", <2>
-    "params": { ... } <3>
+    "lang":   "...",
+    "source" | "id": "...",
+    "params": { ... }
   }
 -------------------------------------
 // NOTCONSOLE
-<1> The language the script is written in, which defaults to `painless`.
-<2> The script itself which may be specified as `source` for an inline script or `id` for a stored script.
-<3> Any named parameters that should be passed into the script.
-
-For example, the following script is used in a search request to return a
-<<script-fields, scripted field>>:
-
-[source,console]
--------------------------------------
-PUT my-index-000001/_doc/1
-{
-  "my_field": 5
-}
-
-GET my-index-000001/_search
-{
-  "script_fields": {
-    "my_doubled_field": {
-      "script": {
-        "lang":   "expression",
-        "source": "doc['my_field'] * multiplier",
-        "params": {
-          "multiplier": 2
-        }
-      }
-    }
-  }
-}
--------------------------------------
-
-[discrete]
-=== Script parameters
 
 `lang`::
 
-    Specifies the language the script is written in.  Defaults to `painless`.
-
+    Specifies the language the script is written in. Defaults to `painless`.
 
 `source`, `id`::
 
-    Specifies the source of the script.  An `inline` script is specified
-    `source` as in the example above. A `stored` script is specified `id`
-    and is retrieved from the cluster state (see <<modules-scripting-stored-scripts,Stored Scripts>>).
-
+    Specifies the source of the script. An `inline` script is specified as `source` in the previous example. A `stored` script is specified by `id` and is retrieved from the cluster state using the `_scripts` endpoint. For
```

Comment:
The old text is fine.
> The script itself which may be specified as source for an inline script or id for a stored script.

Let's add to that by specifying that you can create or delete a stored script by calling the `POST _scripts/{id}` and `DELETE _scripts/{id}` as illustrated below.